x/fees/types/msg.go

```diff
@@ -26,11 +26,16 @@ func NewMsgRegisterFee(
 	withdrawal sdk.AccAddress,
 	nonces []uint64,
 ) *MsgRegisterFee {
+	var withdrawAddr string
```

Comment:
Explicit definition would IMO be preferred, see my reasoning above

```suggestion
	withdrawAddr := ""
```